react_basics/README.md

```diff
@@ -1,5 +0,0 @@
-# React.js basics
```

Comment:
I actually deleted all other folders except the renamed `introduction` to `lesson-0` => we should add a tag to the current master before merging I guess :)